### PR TITLE
fix(DB/gameobject):Remove the Church of the Holy Light with flowers and small mounds

### PR DESCRIPTION
**Pull request by fangshun2004**
Removed East Plaguelands - Church of the Holy Light with flowers and small mounds. Because they are generated during the creature "Rimblat Earthshatter" dialogue event

## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

## Issues Addressed:

- Closes 

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [x] Sniffs (remember to share them with the open source community!)
- [ ] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [x] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1. .go c 54186
2. wait
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by pvp100com** [comment]
Perfect repair.

File: data/sql/updates/pending_db_world/rev_1715349752817459800.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+(@ENTRY * 100, 9, 4, 0, 0, 0, 100, 0, 2000, 2000, 0, 0, 50, 181103, 0, 1, 8, 0, 0, 8, 0, 0, 0, 2277, -5326.86, 88.7085, 5.74214, 'Rimblat Earthshatter - Summon gameobject - Flower (181103) at (2277, -5326.86, 88.7085, 5.74214) and despawn in 0 ms'),
+(@ENTRY * 100, 9, 5, 0, 0, 0, 100, 0, 2400, 2400, 0, 0, 41, 0, 0, 0, 0, 0, 0, 20, 181104, 10, 0, 0, 0, 0, 0, 'Closest gameobject Small Dirt Mound (181104) in 10 yards: Despawn instantly'),
+(@ENTRY * 100, 9, 6, 0, 0, 0, 100, 0, 1500, 1500, 0, 0, 5, 21, 0, 0, 0, 0, 0, 10, 54186, 16135, 0, 0, 0, 0, 0, 'Creature Rayne (16135) with guid 54186 (fetching): Play emote ONESHOT_APPLAUD (21)'),
+(@ENTRY * 100, 9, 7, 0, 0, 0, 100, 0, 3000, 3000, 0, 0, 1, 0, 0, 0, 0, 0, 0, 10, 54186, 16135, 0, 0, 0, 0, 0, 'Creature Rayne (16135) with guid 54186 (fetching): Talk 0 to invoker'),
+(@ENTRY * 100, 9, 8, 0, 0, 0, 100, 0, 58600, 58600, 0, 0, 41, 0, 0, 0, 0, 0, 0, 20, 181103, 10, 0, 0, 0, 0, 0, 'Closest gameobject Flower (181103) in 10 yards: Despawn instantly');
+
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId` = 22 AND `SourceEntry` = 16134 AND `SourceId` = 0;
```
Comment (elthehablo) [inline data/sql/updates/pending_db_world/rev_1715349752817459800.sql:18]: this does not exist in the db as far as I can tell, so why delete?

File: data/sql/updates/pending_db_world/rev_1715349752817459800.sql [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+-- DEL Flower
+DELETE FROM `gameobject` WHERE `id` = 181103;
+
+-- DEL Small Dirt Mound
+DELETE FROM `gameobject` WHERE `id` = 181104;
```
Comment (elthehablo) [inline data/sql/updates/pending_db_world/rev_1715349752817459800.sql:24]:
```suggestion
DELETE FROM `gameobject` WHERE `id` = 181104 AND `guid` = 45965;
```

File: data/sql/updates/pending_db_world/rev_1715349752817459800.sql [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+(@ENTRY * 100, 9, 4, 0, 0, 0, 100, 0, 2000, 2000, 0, 0, 50, 181103, 0, 1, 8, 0, 0, 8, 0, 0, 0, 2277, -5326.86, 88.7085, 5.74214, 'Rimblat Earthshatter - Summon gameobject - Flower (181103) at (2277, -5326.86, 88.7085, 5.74214) and despawn in 0 ms'),
+(@ENTRY * 100, 9, 5, 0, 0, 0, 100, 0, 2400, 2400, 0, 0, 41, 0, 0, 0, 0, 0, 0, 20, 181104, 10, 0, 0, 0, 0, 0, 'Closest gameobject Small Dirt Mound (181104) in 10 yards: Despawn instantly'),
+(@ENTRY * 100, 9, 6, 0, 0, 0, 100, 0, 1500, 1500, 0, 0, 5, 21, 0, 0, 0, 0, 0, 10, 54186, 16135, 0, 0, 0, 0, 0, 'Creature Rayne (16135) with guid 54186 (fetching): Play emote ONESHOT_APPLAUD (21)'),
+(@ENTRY * 100, 9, 7, 0, 0, 0, 100, 0, 3000, 3000, 0, 0, 1, 0, 0, 0, 0, 0, 0, 10, 54186, 16135, 0, 0, 0, 0, 0, 'Creature Rayne (16135) with guid 54186 (fetching): Talk 0 to invoker'),
+(@ENTRY * 100, 9, 8, 0, 0, 0, 100, 0, 58600, 58600, 0, 0, 41, 0, 0, 0, 0, 0, 0, 20, 181103, 10, 0, 0, 0, 0, 0, 'Closest gameobject Flower (181103) in 10 yards: Despawn instantly');
+
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId` = 22 AND `SourceEntry` = 16134 AND `SourceId` = 0;
+
+-- DEL Flower
+DELETE FROM `gameobject` WHERE `id` = 181103;
```
Comment (elthehablo) [inline data/sql/updates/pending_db_world/rev_1715349752817459800.sql:21]:
```suggestion
DELETE FROM `gameobject` WHERE `id` = 181103 AND `guid` = 45964;
```

**Review by elthehablo — CHANGES_REQUESTED** [review]

**Review by elthehablo — APPROVED** [review]
Thanks for the pr once again

**Pull request merged**